### DISCUSSION TITLE
Allow querying users by multiple permissions

### DISCUSSION THREAD

**Discussion by goraxan** [craftcms/cms#8952] [Ideas]
### Description
Currently you can query users by a specific permission as explained in https://docs.craftcms.com/api/v3/craft-elements-db-userquery.html#can
Would it be possible to create a query to create more complex queries like fetching users containing any of the specified permisions?
Similar to how you query by dateCreated (https://craftcms.com/docs/3.x/users.html#datecreated)
```
\craft\elements\User::find()
    ->can(['or', "accessCp", "editUsers"])
    ->all();
```
### Additional info
Craft version: 3.5.7

**Comment by brandonkelly** [comment]
Until this becomes possible, you could do:

```php
$ids = array_unique(array_merge(
  User::find()->can('accessCp')->ids(),
  User::find()->can('editUsers')->ids()
));
$users = User::find()->id($ids)->all();
```

**Comment by goraxan** [comment]
That'll work for now, thanks!
